content/manual/os/dojo-tools.md

```diff
@@ -415,3 +497,2958 @@ None.
+
+```
+> :azimuth|listen ~[~sampel ~palnet] %ship ~wet
+>=
+```
+
+---
+
+### `-azimuth-load`
+
+Refetch and load Azimuth snapshot.
+
+#### Example
+
+```
+> -azimuth-load
+ship: loading azimuth snapshot (106.177 points)
+[%eth-watcher 'overwriting existing watchdog on' /azimuth]
+ship: processing azimuth snapshot (106.177 points)
+```
+
+---
+
+### `+azimuth-sources`
+
+List all Azimuth sources.
+
+This will print a
+[`state-eth-node:jael`](https://developers.urbit.org/reference/arvo/jael/data-types#state-eth-node)
+structure. Its contents is mostly other ships who are sources for updates about
+moons, but it will also include `%azimuth`.
+
+#### Example
+
+```
+> +azimuth-sources
+[ top-source-id=0
+  sources={}
+  sources-reverse={}
+  default-source=0
+  ship-sources={}
+  ship-sources-reverse={}
+]
+```
+
+---
+
+### `:azimuth|watch`
+
+Change node URL and network for Azimuth.
+
+#### Arguments
+
+```
+cord ?(%mainnet %ropsten %local %default)
+```
+
+The first argument is the note URL in a cord like
+`'http://eth-mainnet.urbit.org:8545' `. The second argument specifies the
+network.
+
+#### Example
+
+```
+> :azimuth|watch 'http://eth-mainnet.urbit.org:8545' %default
+>=
+```
+
+---
+
+## CLI Apps
+
+These commands are for managing the dojo and other CLI apps.
+
+### `:dojo|acl`
+
+Show which ships you've allowed remote access to your dojo.
+
+#### Example
+
+```
+~zod:dojo> :dojo|allow-remote-login ~bus
+>=
+
+~zod:dojo> :dojo|acl
+>=
+{~bus}
+```
+
+---
+
+### `:dojo|allow-remote-login`
+
+Allow a ship to `|link` your dojo.
+
+#### Arguments
+
+The ship you want to allow remote access.
+
+```
+ship
+```
+
+#### Example
+
+Allow ~bus to link ~zod's dojo:
+
+```
+~zod:dojo> :dojo|allow-remote-login ~bus
+>=
+
+~zod:dojo> :dojo|acl
+>=
+{~bus}
+```
+
+Link ~zod's dojo on ~bus:
+
+```
+~bus:dojo> |link ~zod %dojo
+>=
+[linked to [p=~zod q=%dojo]]
+
+~zod:dojo>
+```
+
+---
+
+### `:dojo|wipe`
+
+Clear the dojo's subject.
+
+This will clear all values pinned to the dojo's subject with commands like `=foo
+42`.
+
+#### Example
+
+```
+> =foo 42
+
+> foo
+42
+
+> :dojo|wipe
+>=
+
+> foo
+-find.foo
+dojo: hoon expression failed
+```
+
+---
+
+### `:dojo|revoke-remote-login`
+
+Revoke permission for a remote ship to `|link` your dojo.
+
+#### Arguments
+
+The ship whose permission you wish to revoke.
+
+```
+ship
+```
+
+#### Example
+
+```
+~zod:dojo> :dojo|allow-remote-login ~bus
+>=
+
+~zod:dojo> :dojo|acl
+>=
+{~bus}
+
+> :dojo|revoke-remote-login ~bus
+>=
+
+> :dojo|acl
+>=
+{}
+```
+
+---
+
+### `|link`
+
+Connect a local or remote CLI app.
+
+#### Arguments
+
+```
+ship term
+```
+
+The `ship` is optional, it's only necessary if the app is on a remote ship. The
+`term` is mandatory, and is the name of the CLI app to connect.
+
+#### Example
+
+Connect to the dojo on a remote ship (this is only possible if you have
+permission):
+
+```
+|link ~sampel-palnet %dojo
+>=
+[linked to [p=~sampel-palnet q=%dojo]]
+~sampel-palnet:dojo>
+```
+
```

Comment:
Are there any others in common use these days?  May be worth listing if so.  %herd

Reply:
Also mention that this is common with moons.